cmake/dist/startup_macosx.sh.in

```diff
@@ -16,14 +16,14 @@ tell application "Terminal" to set size of front window to {$w, ${dim[1]}}
 EOF
   export PATH="${BUNDLE_RESOURCES}/@GMT_BINDIR@:${PATH}"
   export PROJ_LIB="${BUNDLE_RESOURCES}/share/proj"
-  export GS_LIB="${BUNDLE_RESOURCES}/share/ghostscript/9.53.3/Resource/Init"
+  export GS_LIB="${BUNDLE_RESOURCES}/share/ghostscript/@GMT_CONFIG_GS_VERSION@/Resource/Init"
   export MAGICK_CONFIGURE_PATH=${BUNDLE_RESOURCES}/lib/GraphicsMagick/config
 
   function gmt () { "${BUNDLE_RESOURCES}/@GMT_BINDIR@/gmt" "$@"; }
   export -f gmt
   unset DYLD_LIBRARY_PATH
   gmt
-  echo -e "Note 1: If you want to use GMT outside of this terminal or in scripts, then follow these steps:\n        a) export GMTHOME=${BUNDLE_RESOURCES}\n        b) add \$GMTHOME/bin to your path\n        c) export PROJ_LIB=\$GMTHOME/share/proj\n        d) export GS_LIB=\$GMTHOME/share/ghostscript/9.53.3/Resource/Init\n        e) export MAGICK_CONFIGURE_PATH=\$GMTHOME/lib/GraphicsMagick/config"
+  echo -e "Note 1: If you want to use GMT outside of this terminal or in scripts, then follow these steps:\n        a) export GMTHOME=${BUNDLE_RESOURCES}\n        b) add \$GMTHOME/bin to your path\n        c) export PROJ_LIB=\$GMTHOME/share/proj\n        d) export GS_LIB=\$GMTHOME/share/ghostscript/@GMT_CONFIG_GS_VERSION@/Resource/Init\n        e) export MAGICK_CONFIGURE_PATH=\$GMTHOME/lib/GraphicsMagick/config"
```

Comment:
Sorry for commenting on a merged PR.

For GMT 6.2.0, users need to set `GS_LIB` to
```
export GS_LIB=$GMTHOME/share/ghostscript/9.53.0/Resource/Init
```
when users upgrade their GMT 6.2.0 to GMT 6.3.0, they have to change `GS_LIB` to 
```
export GS_LIB=$GMTHOME/share/ghostscript/9.55.0/Resource/Init
```

I believe it's very inconvenient and users may forget it. Can we remove the GS version from the path?

Reply:
Possibly, I will have a look at the build-release.sh and sub-scripts.  I agree it is dumb but we initially just maintained what Ghostscript does on any system.  But setting GS_LIB should let us do what we want.